benchmark_runner/krkn_hub/krknhub_workloads.py

```diff
@@ -44,6 +44,12 @@ def finalize_workload(self):
         """
         if self._enable_prometheus_snapshot:
             self.end_prometheus()
+        if self._google_drive_shared_drive_id:
+            self.upload_run_artifacts_to_google_drive()
+        elif self._endpoint_url and not self._google_drive_shared_drive_id:
```

Comment:
Don't need the "and not self.__google_drive_shared_drive_id" because that's already covered by the previous case.  Otherwise everything is good.

Reply:
fixed